data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java

```diff
@@ -455,11 +481,12 @@ public Reader getCharacterStream(String columnName) {
     }
 
     public BigDecimal getBigDecimal(int columnIndex) {
```

Comment:
I'd love to move this method to where it belongs, but I'll wait for another PR so I don't make this PR too difficult to follow.